_posts/release/2020-01-19-accumulo-2.1.0.md

```diff
@@ -246,8 +246,11 @@ are in a new [repo] [vfs_system_classloader] and can now be specified using syst
 
 Consolidated and normalized thread pool and thread creation. All threads created through this code path
 will have an UncaughtExceptionHanlder attached to it that will log the fact that the Thread encountered
-an uncaught Exception and is now dead. When an Error is encountered it will attempt to print a message
-to stderr then terminate the VM using Runtime.halt. {% ghi 1808 %} {% ghi 1818 %}
+an uncaught Exception and is now dead. When an Error is encountered in a server process it will attempt
+to print a message to stderr then terminate the VM using Runtime.halt. On the client side the default
+UncaughtExceptionHandler will only log the Exception/Error in the client and does not terminate the VM.
+Additionally the user has the ability to set their own UncaughtExceptionHandler implementation on the
```

Comment:
```suggestion
an uncaught Exception and is now dead. When an Error is encountered in a server process, it will attempt
to print a message to stderr then terminate the VM using Runtime.halt. On the client side, the default
UncaughtExceptionHandler will only log the Exception/Error in the client and does not terminate the VM.
Additionally, the user has the ability to set their own UncaughtExceptionHandler implementation on the
```